utils/data_processing.py

```diff
@@ -0,0 +1,44 @@
+"""Process and format the mesh data for our model"""
+
+from datatypes.graph import EdgeSet
+from datatypes.graph import Graph
+
+import numpy as np
+
+
+def _triangles_to_edges(faces):
```

Comment:
Are we completely sure that cells are defined by triangles? Can they be quadrilaterals?

Reply:
I worked with triangle cells because this is the objects I was provided with, but we can adapt the code later on to consider other shapes

Reply:
Have you checked the "bird.obj"? It seems to be quadrilateral.

To be more general, the appropriate way is to use the adjacency matrix and generate the edges from there. With Graph Networks we are not conflicted with type of mesh as in classical simulators, thus we can go for a more general methodology.

Reply:
I could not load bird.obj with meshio because there is a problem with the its texture, but I think .obj files are always triangle meshes.

Reply:
For the adjacency matrix, we rather encode the edges as a list here. The high number of nodes and low connectivity probably justify it. With quadrilateral meshes we should be able to easily add the 4th edge to this list.

Reply:
With pyvista you can read the mesh and see that it is quadrilateral. Obj supports polygonal meshes, but it might be the encoding that meshio is not reading correctly! We will make sure that with the dataset we can read everything independently of the mesh type.

With general meshes we can encode it has a list as well!